src/agent/tools/tier2_advanced.py

```
@@ -1058,10 +1058,32 @@ def _build_synthesis_prompt(
+                system_prompt += " Analyze patterns, themes, and deeper insights."
+
+            response = self.llm_provider.create_message(
+                messages=[{"role": "user", "content": prompt}],
+                max_tokens=2000,  # Allow long synthesis
+                temperature=0.3,  # Lower temperature for factual synthesis
+                system=system_prompt,
+                tools=[]
+            )
+            return response.content[0].text
+            
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)